torch/_inductor/cudagraph_trees.py

```diff
@@ -820,11 +845,7 @@ def __init__(
         ]
 
         self.static_input_data_ptrs: InputList[Optional[int]] = [
-            (
-                inputs[i].data_ptr()
-                if isinstance(inputs[i], torch.Tensor) and i in self.static_input_idxs
-                else None
-            )
+            maybe_get_static_data_ptr(i, inputs, self.static_input_idxs)
```

Comment:
nit - this function is only used once.. could we inline it instead of defining elsewhere ?